test/testinput/bufr_query_python_test.py

```diff
@@ -38,7 +38,8 @@ def test_basic_query():
     assert len(rad_all.shape) == 2
 
     datetimes = r.get_datetime('year', 'month', 'day', 'hour', 'minute', 'second')
-    assert datetimes[5] == np.datetime64('2020-10-26T21:00:00')
+    assert datetimes[5] == np.datetime64('2020-10-26T21:00:01')
+    assert datetimes.fill_value == np.datetime64('1970-01-01T00:00:00')
```

Comment:
Am I understanding this to mean the fill_value is 0?

Reply:
You shouldn't need to use the fill_value directly unless you are writing datetime data to a netcdf file (see **test/testinput/bufr_query_python_to_ioda_test.py**). The fill values for datetimes are 0 seconds after Jan 1, 1970 (0:00).

Reply:
we'll need to keep talking about this as a `0` valid and system may be run at or before the ctime epoch

stick a pin in this one ;0)

Reply:
@BenjaminRuston The other possible **fill value** we might consider is the max number of int64 (seconds since the epoch). This will give you a date in the far future... Would this be preferable?